frontend/src/components/form/api/ApiWrapper.vue

```diff
@@ -5,7 +5,8 @@ Wrapper component for form components to save data back to API
 <template>
   <v-form
     inline
-    :class="{'d-flex api-wrapper--savable':!autoSave && !readonly,'my-4':!noMargin}">
+    :class="{'d-flex api-wrapper--savable':!autoSave && !readonly}"
```

Comment:
Auf dem Slot sollte wohl noch `:noMargin="autoSave && readonly"``
dann hat man das Problem mit den Buttons nicht mehr. Zusätzlich sollte falls nicht `noMargin` auf die Api Komponente gesetzt wird und aber die Komponente `autoSave && !readonly`, hier noch `my-4`  auf das v-Form gesetzt werden.

Sorry ist jetzt ein bisschen umständlich geschrieben.

Reply:
Auf dem Slot selber geht leider nicht. Aber habs wieder zurück in den Wrapper genommen. Und noch etwas umgebaut (vor allem auch für Textarea und Checkbox).